### ISSUE TITLE
Mounting a Synced Folder for a theme -> Broken Theme: Stylesheet is missing.

### ISSUE THREAD

**Issue by jmbarlow** [Varying-Vagrant-Vagrants/VVV#751]
I'm updating my development environment on a new Windows 10 box, and I've got a peculiar problem with Synced Folders. I've adding a `Customfile` containing one line to map a local directory containing a wordpress theme into a folder inside the default VVV installation.

```
config.vm.synced_folder "X:/gitRepository/someCustomTheme", "/vagrant/www/wordpress-default/wp-content/themes/someCustomTheme", :owner => "vagrant", :mount_options => []
````
Looking at the Wordpress Admin screen, the theme is listed as "Broken", with the error `"Stylesheet is missing"`. And yet, if I `vagrant ssh` into the box, I can see the synced folder and its contents (which include a style.css); I can create files in the local system or host; the ownership and access permissions are 777. If I disable the synced folder, and just copy the files over, everything works just fine.

I'm sure I'm missing something blindingly obvious, and I've had a similar setup working before. Anyone got any ideas?

Versions: Vagrant 1.7.4,  Virtual Box 5.0.4, Latest VVV

**Comment by ezekg** [comment]
It's probably something to do with the permissions for the synced folder. To allow Vagrant full access, try specifying the following in your `:mount_options` array:

```ruby
config.vm.synced_folder "path/to/local", "path/to/vm", :mount_options => ["dmode=777", "fmode=777"]
```

**Comment by jmbarlow** [comment]
No joy yet. It's not a major issue as I can just copy files around, but it's just odd. To recap:
1.Vagrant is mounting the synced folder just fine, and as far as the guest OS is concerned, everything is good. I can edit and change the file on the host or the guest, and all the permissions, owners and group owners are as expected. So shell access is all good, which suggests the file system is working just fine.
2. Nginx won't serve files from the synced folders. I thought it might be an Nginx issue, so changed that to run as user "vagrant". No difference.
3. PHP might be the problem. If I create a simple test script running from the document root of, say, wordpress-default with a simple include like 'include("./wp-content/themes/twentyfifteen/style.css");' then it serves the content  fine, but not if there is a synced folder in the path - PHP can't find the file.

Really odd. I don't think it's a VVV problem per-se, but I've managed to configure Windows 10 in such a way that I'm able to confuse the virtual filesystem. Not sure if there's a badge I can get for that.

**Comment by onlyskyguy** [comment]
Having what looks like the same problem, although not seeing the files at all:

I added a "config.vm.synced_folder" line to Customfile that should link a directory on my local machine to a theme folder inside one of the vvv WP installs. 
```
config.vm.synced_folder "/PATH/TO/THEMEFILES", "/vagrant/www/WPSITE/htdocs/wp-content/themes/THEMEFILES/", :owner => "www-data", :mount_options => [ "dmode=775", "fmode=774" ]
```
It looks like the vm.synced directory gets created fine but it's empty (I looked at it with vagrant ssh too and it's empty there as well). When vvv boots up with vagrant up I see this:

```
==> default: Mounting shared folders...
... 
default: /vagrant/www/WPSITE/htdocs/wp-content/themes/THEMEFILES => /PATH/TO/THEMEFILES

```

My SSD is encrypted, but would that have any effect on this since it's all running off of the same SSD?

My scenario is like this: I have a theme I'm developing using gulp, so my finished theme files get output into a directory called "build." I would like to link that directory to the themes folder on the site I'm developing in VVV. I'd rather not have all my development files live in the themes folder for the vagrant virtual host... 

Is there some permissions problem? The new theme directory in /srv/www... has the following permissions:

```
drwxrwxr-x 1 www-data vagrant 4096
```

How can I do this?  Thanks for any help!
